shaders/src/common_types.fs

```diff
@@ -34,3 +34,8 @@ struct ShadowData {
     float bulbRadiusLs;
     float nearOverFarMinusNear;
 };
+
+struct BoneData {
+    highp mat3x4 transform;    // bone transform is mat4x3 stored in row-major (last row [0,0,0,1])
+    highp uvec4 cof;           // 8 first cofactor matrix of transform's upper left
```

Comment:
Could these be floats to avoid int<>float conversions on the GPU and avoiding the int pipeline?

Reply:
no because we can't store fp16 floats in ubos. we can only pack two fp16 into one `int`.

Reply:
Ok makes sense